src/SeleniumLibrary/keywords/formelement.py

```diff
@@ -192,72 +193,78 @@ def select_radio_button(self, group_name, value):
 
     @keyword
     def choose_file(self, locator, file_path):
-        """Inputs the `file_path` into file input field found by `locator`.
+        """Inputs the ``file_path`` into file input field ``locator``.
 
         This keyword is most often used to input files into upload forms.
-        The file specified with `file_path` must be available on the same host
-        where the Selenium Server is running.
+        The file specified with ``file_path`` must be available on machine
+        where tests are executed.
```

Comment:
This actually might not be true, at least there is an issue about it: https://github.com/robotframework/SeleniumLibrary/issues/449

Also I have recollection that Choose File works also with when using selenium gird.

Reply:
The code [explicitly validates](https://github.com/robotframework/SeleniumLibrary/blob/master/src/SeleniumLibrary/keywords/formelement.py#L194) that the file exists on the machine where the library is installed. That is generally also the machine where tests are executed.

Reply:
I have this feeling that it should work, let me try it out and let's see what happens.